Clarify missing default claims in JWT templates (#2286)
clerkinc/clerk-docs: docs/backend-requests/jwt-templates.mdx

In every generated token, there are certain claims that are automatically included and cannot be overridden by templates. Clerk calls these "default claims" and you can learn more about them in the [Session tokens](/docs/backend-requests/resources/session-tokens) reference documentation.

For custom JWTs, Clerk automatically includes the following default claims:

```json
{
  // default claims, included automatically
  "azp": "http://localhost:3000",
  "exp": 1639398300,
  "iat": 1639398272,
  "iss": "https://clean-mayfly-62.clerk.accounts.dev",
  "jti": "10db7f531a90cb2faea4",
  "nbf": 1639398220,
  "sub": "user_1deJLArSTiWiF1YdsEWysnhJLLY"
}
```

> However, other default claims - such as `sid` (session ID) - are only included in session-bound tokens, not in custom JWTs. Custom JWTs are not inherently tied to a session. Instead, they're designed to be flexible tokens that include the default claims listed above, along with any additional claims you explicitly set in your JWT template. For this reason, session-tied claims like `sid`, `v`, `pla` or `fea` cannot be included in custom JWTs.

Reviewer: VERY small nit but oxford comma's !

```suggestion
However, other default claims - such as `sid` (session ID) - are only included in session-bound tokens, not in custom JWTs. Custom JWTs are not inherently tied to a session. Instead, they're designed to be flexible tokens that include the default claims listed above, along with any additional claims you explicitly set in your JWT template. For this reason, session-tied claims like `sid`, `v`, `pla`, or `fea` cannot be included in custom JWTs.
```

> If you need to generate a token that includes both session-bound data (like `sid`) and any additional claims, the recommended approach is to use a custom session token instead of a JWT template. You can do this by adding custom claims to the session token using this [guide](/docs/backend-requests/custom-session-token).

Reviewer: ```suggestion
If you need to generate a token that includes both session-bound data (like `sid`) and any additional claims, the recommended approach is to use a custom session token instead of a JWT template. See the [guide on customizing your Clerk session token](/docs/backend-requests/custom-session-token).
```
